### PR TITLE
Feature/task update tags

### PR DESCRIPTION
**Pull request by alexandre-snr**
This PR adds the 'update-tags' task.

This task uses the lastest daily usage reports to generate tagging reports.
These tagging reports contains resource's type, id, region, url and the tags associated with this resource.

The tagging reports are stored in a new ES index called *-tagging-reports for further usage.
All supported resources are implemented (still need an example URL to build the Elasticache url model)

Tests were made with the used mapping and it should be OK for the usage we would have (most used tags, resources without a specified tag ...)

### CODE REVIEW

File: tagging/mappings.go
Comment (jklakosz) [inline tagging/mappings.go:21]: Missing `\n` at the end of line.

**Review by jklakosz — COMMENTED** [review]
Good for me except the missing `\n`. Waiting for @simonmeyerrr approval.

**Review by jklakosz — APPROVED** [review]
Good for me, waiting for @simonmeyerrr approval.

File: db/migration/0044_add_aws_account_update_tags_job.sql
Comment (simonmeyerrr) [inline db/migration/0044_add_aws_account_update_tags_job.sql]: Add a newline please

File: db/schema.sql
Comment (simonmeyerrr) [inline db/schema.sql]: Add a new line please

File: db/schema.sql
Comment (simonmeyerrr) [inline db/schema.sql]: not aligned

File: tagging/ebs/process.go
Comment (simonmeyerrr) [inline tagging/ebs/process.go]: If you want to log some values, use a map[string]interface{} as second parameter instead of using fmt.Sprintf

File: tagging/ebs/process.go
Comment (simonmeyerrr) [inline tagging/ebs/process.go]: If you want to log some values, use a map[string]interface{} as second parameter instead of using fmt.Sprintf

File: tagging/ebs/process.go
Comment (simonmeyerrr) [inline tagging/ebs/process.go]: If you want to log some values, use a map[string]interface{} as second parameter instead of using fmt.Sprintf

File: tagging/ebs/es.go
Comment (simonmeyerrr) [inline tagging/ebs/es.go]: please split this function

File: tagging/mappings.go
Comment (simonmeyerrr) [inline tagging/mappings.go]: bad indentation

**Review by simonmeyerrr — CHANGES_REQUESTED** [review]
Apply corrections I asked for `tagging/ebs/process.go` for every other similar files.

**Review by simonmeyerrr — APPROVED** [review]

File: tagging/ec2Ri/process.go
Comment (simonmeyerrr) [inline tagging/ec2Ri/process.go]: Use the structure from the original package

File: tagging/ec2Ri/process.go
Comment (simonmeyerrr) [inline tagging/ec2Ri/process.go]: Use the index name from the original package

**Review by simonmeyerrr — CHANGES_REQUESTED** [review]
Make sure to use good Ids everywhere

File: server/taskUpdateTags.go
Comment (simonmeyerrr) [inline server/taskUpdateTags.go]: rename variable into  `aaId` to be consistent with the rest of the code

File: server/taskUpdateTags.go
Comment (simonmeyerrr) [inline server/taskUpdateTags.go]: please follow the format of the function `ingestDataForAccount`

File: tagging/ebs/es.go
Comment (simonmeyerrr) [inline tagging/ebs/es.go]: use the format with only one variable to be consistent with the rest of the code

File: tagging/ebs/process.go
Comment (simonmeyerrr) [inline tagging/ebs/process.go]: You can remove the first region for all urls before `.console.`

**Review by simonmeyerrr — CHANGES_REQUESTED** [review]

**Review by simonmeyerrr — APPROVED** [review]

**Pull request merged**
